README.md

```diff
@@ -50,11 +50,30 @@ These are the steps to publish either a new release or a hotfix to an existing r
 
 4. In `scripts`, run `python do_all_things.py -v VERSION`. The script assumes you used `npm` to install. See example 3 if you use `yarn`.
 
-   **Example 1**: `python do_all_things.py -v 26.0.0`. This command builds version 26.0.0 of the docs and latest.
-
-   **Example 2**: If you already have Docusaurus 2 installed, skip the installation by specifying the flag `--skip-install`. For example, `python do_all_things.py -v 26.0.0 --skip-install`
-
-   **Example 3**: If you want to use yarn instead of npm, specify the flag `--yarn`. For example, `python do_all_things.py -v 26.0.0 --skip-install --yarn`
+    **Example 1**: This command builds version 27.0.0 of the docs and latest.
+    ```
+    python do_all_things.py -v 27.0.0
+    ```
+
+    **Example 2**: If you already have Docusaurus 2 installed, skip the installation by specifying the flag `--skip-install`.
+    ```
+    python do_all_things.py -v 27.0.0 --skip-install
+    ```
+
+    **Example 3**: If you want to use yarn instead of npm, specify the flag `--yarn`.
+    ```
+    python do_all_things.py -v 27.0.0 --yarn
+    ```
+
+    **Example 4**: If you have apache/druid in a completely different place (see note above), specify it using the `--source` flag.
+    ```
+    python do_all_things.py -v 27.0.0 --source /my/path/to/apache/druid
+    ```
+
+    **Example 5**: If you want to skip all the source docs (don't copy them from apache/druid), use the `--no-docs` flag.
```

Comment:
```suggestion
    **Example 5**: If you want to use the Markdown files already in this repo because you don't need to publish doc changes, use the `--no-docs` flag. The script will use the docs already in `docs/latest` to build the site. Use this if you need to republish the site to update the homepage widgets or other non-doc website pages.
```

* Probably update the updating the homepage widgets section section to mention that if all you need to do is update the data for the homepage widgets, use the `--no-docs` flag when running `do_all_the_things` script.
* Remove `27.0.0` from the example since the intent is that this flag is primarily used for updating the website pages, which means we only need to build latest.